src/components/ReactSlider/ReactSlider.jsx

```diff
@@ -222,25 +222,26 @@ class ReactSlider extends React.Component {
 
         /**
          * Callback called before starting to move a thumb. The callback will only be called if the
-         * action will result in a change. The function will be called with one argument,
-         * the initial value(s).
+         * action will result in a change. The function will be called with two arguments, the first
+         * being the new value(s) the second being thumb index.
```

Comment:
Can you change "new value(s)" back to "initial value(s)"?

Reply:
done

Reply:
Looks like this is still saying "new" instead of "initial".

Reply:
uhh sorry missed this one, fixed ;)